Gemfile.lock

```diff
@@ -42,17 +42,52 @@ GEM
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
```

Comment:
Why is tzinfo in here?

Reply:
'database_cleaner' seems to depend on it:
https://rubygems.org/gems/database_cleaner/versions/1.5.3